tests/test_encoding.py

```diff
@@ -85,7 +85,6 @@ def test_encode_coded_const_infer_order(self) -> None:
             admin_data=None,
             sdgs=[],
             parameters=NamedItemList([param1, param2]),
-            byte_size=None,
```

Comment:
Why was `byte_size=None,` removed?

Reply:
possible to move this change to its own PR? this particular change is too noisy in this PR

Reply:
that's probably possible, but as I see it, most of the point of this PR is not having to derive `Request` and `Response` from `BasicStructure` anymore...